app/src/main/java/com/pvp/app/ui/screen/goals/GoalScreen.kt

```diff
@@ -238,9 +238,10 @@ fun GoalCard(
                     style = MaterialTheme.typography.bodyMedium,
                     modifier = Modifier.padding(start = 6.dp),
                     textAlign = TextAlign.Left,
-                    text = "You're average " + when (goal.monthly) {
-                        true -> "monthly steps: %d".format((monthSteps / 30))
-                        false -> "weekly steps: %d".format((monthSteps / 30 * 7))
+                    text = if (goal.monthly) {
```

Comment:
Not _only here_, but the whole screen must be localized. Lines 277, 297 included.